### DISCUSSION TITLE
Bitstamp exchange?

### DISCUSSION THREAD

**Discussion by RN88R** [taoteh1221/Slideshow_Crypto_Ticker#8] [Q&A] [answered]
First of all, thank you for this awesome ticker code!

Im new to coding and try to learn. 
I bought a rpi3 and try different projects from github.
I installed your ticker with the auto installer. (no tft screen)
Now i want to change the standard coinbase exhange to bitstamp.
I simply removed kraken en changed it to:

// Bitstamp markets (set to '' to disable)
exchange_markets['bitstamp'] = 'BTCEUR'; 

When i reboot it says:  BTC (connecting...)
I prefer euro and tried: BTCEUR btreur BTC-EUR btc-eur BTC/EUR
Where can i find the correct setting or code? 
Kind regards

**Comment by taoteh1221** [comment]
Your welcome, and thanks for the compliments! :) There is no support for Bitstamp yet, but it's in the TODO list to add at some point. I may have time this weekend, when I look into adding kucoin support https://github.com/taoteh1221/DFD_Crypto_Ticker/discussions/6

**Comment by taoteh1221** [comment, marked as answer]
Bitstamp support enabled: https://github.com/taoteh1221/Slideshow_Crypto_Ticker/releases

Enjoy! :)
